tools/validate_definitions.py

```
@@ -0,0 +1,366 @@
+
+REQUIRED_KEYS = [
+    "FIELDS",
+    "LEDGER_ENTRY_TYPES",
+    "TRANSACTION_RESULTS",
+    "TRANSACTION_TYPES",
+    "TYPES",
+]
+
+
+def _exec(cmd: str) -> str:
+    result = subprocess.run(
```

Ruff (0.15.14)

[error] `subprocess` call with `shell=True` identified, security issue

(S602)